README.md

```diff
@@ -1 +1,149 @@
-# OOP_Libaray
\ No newline at end of file
+# OOP_Libaray
+<a name="readme-top"></a>
+ 
+
+<div align="center"> 
+  <img src="murple_logo.png" alt="logo" width="40"  height="auto" />
+  <br/>  
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
```

Comment:
- Hey, please, since you did not deploy the app, please, let's kindly remove this section from the readme to make the readme more professional.